cms/templates/course_outline.html

```diff
@@ -193,16 +193,29 @@ <h2 class="sr">${_("Course Outline")}</h2>
                 <h3 class="title-3">${_("Creating your course organization")}</h3>
                 <p>${_("You add sections, subsections, and units directly in the outline.")}</p>
                 <p>${_("Create a section, then add subsections and units. Open a unit to add course components.")}</p>
+            </div>
+            <div class="bit">
                 <h3 class="title-3">${_("Reorganizing your course")}</h3>
                 <p>${_("Drag sections, subsections, and units to new locations in the outline.")}</p>
+                <div class="external-help">
+                    <a href="${get_online_help_info('outline')['doc_url']}" target="_blank" class="button external-help-button">${_("Learn more about the course outline")}</a>
+                </div>
+            </div>
+            <div class="bit">
                 <h3 class="title-3">${_("Setting release dates and grading policies")}</h3>
                 <p>${_("Select the Configure icon for a section or subsection to set its release date. When you configure a subsection, you can also set the grading policy and due date.")}</p>
-                <h3 class="title-3">${_("Changing the content students see")}</h3>
-                <p>${_("To publish draft content, select the Publish icon for a section, subsection, or unit.")}</p>
-                <p>${Text(_("To hide content from students, select the Configure icon for a section, subsection, or unit, then select {em_start}Hide from students{em_end}.")).format(em_start=HTML("<strong>"), em_end=HTML("</strong>"))}</p>
+                <div class="external-help">
+                    <a href="${get_online_help_info('grading')['doc_url']}" target="_blank" class="button external-help-button">${_("Learn more about grading policy settings")}</a>
+                </div>
             </div>
-            <div class="bit external-help">
-                <a href="${get_online_help_info(online_help_token())['doc_url']}" target="_blank" class="button external-help-button">${_("Learn more about the course outline")}</a>
+            <div class="bit">
+                <h3 class="title-3">${_("Changing the content learners see")}</h3>
+                <p>${_("To publish draft content, select the Publish icon for a section, subsection, or unit.")}</p>
+                <p>${Text(_("To make a section, subsection, or unit unavailable to learners (including grade calculations), select the Configure icon for a section, subsection, or unit, then select {em_start}Hide from learners{em_end}.")).format(em_start=HTML("<strong>"), em_end=HTML("</strong>"))}</p>
+                <p>${Text(_("To hide content (but not grades) from learners after a due date, select the Configure icon for a subsection, then select {em_start}Hide contents but not grades from learner after due date{em_end}.")).format(em_start=HTML("<strong>"), em_end=HTML("</strong>"))}</p>
```

Comment:
To hide the content of a subsection from learners after the subsection due date has passed, select the Configure icon for a subsection, then select ```{em_start}```Hide Content After Due Date```{em_end}```. Grades for the subsection remain included in grade calculations.